### ISSUE TITLE
Fixes for schema broken by commit 276150255d1843021dcd8b47809c0f79a6660517

### ISSUE THREAD

**Issue by pebender** [alexa-samples/alexa-smarthome#189]
Using git bisect, this is the first commit where the schema is broken according to both https://www.jsonschemavalidator.net and ajv (https://github.com/ajv-validator/ajv)

**Comment by pebender** [comment]
I have created 3 patches that make the schema validate against draft-04, draft-06, draft-07 and draft-2019-09. Since the schema is autogenerated and I don't have access to the generator, I decided patches were the best approach to convey what might need to change in the generator. The patches can be found here https://github.com/pebender/alexa-for-lg-webos-tv/tree/df653c851d5e088ed32f44f7e2179a24e8f35974/src/common/alexa

**Comment by aszk** [comment]
Could you tell me what is the issue?

**Comment by aszk** [comment]
Closing due to inactivity.

**Comment by pebender** [comment]
The problem is that the generated JSON in [alexa_smart_home_message_schema.json](https://github.com/alexa-samples/alexa-smarthome/blob/276150255d1843021dcd8b47809c0f79a6660517/validation_schemas/alexa_smart_home_message_schema.json) is broken. As I mentioned above in my first comment, it fails with two different validators. As a result of it not working with the AJV validator is that it cannot be use to easily validate discovery messages. The patches I referenced above are what I had to change to get it to work.
